### DISCUSSION TITLE
Need more detailed starter guide

### DISCUSSION THREAD

**Discussion by Arctomachine** [TanStack/router#739] [General]
I started learning this tool and documentation seems not complete and not beginner friendly. It describes many aspects of tool, which might be good for people experienced with it.
But what it lacks is more straightforward instruction on how to combine all individual parts into working app, which is essential for people trying this tool for first time.

**Comment by Benanna2019** [comment]
I think it could be tailored a bit more. My thought was to take the quickstart section but break it down and make it more of a quickstart with installation and creating a first route. 

The there could be subsequent sections for adding a loader and so on. Is this what you were thinking @Arctomachine? A way to start a project from the ground up with?

**Reply by Arctomachine** [reply]
Yes. Maybe even short tutorial on making small yet complete app (in different frameworks?) with heavy focus on router.

**Reply by Benanna2019** [reply]
This is a very basic guide but I think it's a good walkthrough of getting started with the router

Let me know what you think - https://www.benapatton.dev/articles/getting-started-with-tanstack-router
